Source/WebCore/platform/Pasteboard.h

```diff
@@ -72,6 +72,8 @@ class FragmentedSharedBuffer;
 
 struct SimpleRange;
 
+static constexpr unsigned pasteboardExpireTime = 8*60;
```

Comment:
Strong typing may help here (with the explicit time unit)!

```cpp
static constexpr auto pasteboardExpireTime { 8_min };
```

Reply:
and then you could use it like

```objc
[m_pasteboard _setExpirationDate:[NSDate dateWithTimeIntervalSinceNow:pasteboardExpireTime.seconds()]];
```

Reply:
also totally optional language nit, but I would call it `duration` over `time` --> `pasteboardExpiryDuration`?